app/src/main/java/com/appculinaryrecipes/Recipe.java

```diff
@@ -1,31 +1,124 @@
 package com.appculinaryrecipes;
 
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
 public class Recipe {
     private String id;
-    private String title;
-    private String imageURL;
+    private String rating;
```

Comment:
ale tutaj to chyba ja już zmieniałem tą klase, dlatego będą konflikty na pewno

Reply:
Konfikty sa rozwiązane

Reply:
Konfikty sa rozwiązane